modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2537,7 +2537,8 @@ export class Wallet implements IWallet {
     } else {
       // Rewrite tokens into recipients for buildTransaction
       buildParams.recipients = params.enableTokens.map((token) => {
-        const address = token.address || this._wallet.coinSpecific?.baseAddress;
+        const address =
+          token.address || this._wallet.coinSpecific?.baseAddress || this._wallet.coinSpecific?.rootAddress;
```

Comment:
What is the difference between `baseAddress` and `rootAddress`?

Reply:
Would be good to add this as a comment in the code.

Reply:
done!